src/main.c

```diff
@@ -13,13 +13,19 @@ int main(int argc, char *argv[])
     appContext.currentTileNumber = EMPTY_TEXTURE_NUMBER;
     appContext.isErasing = 0;
 
+    initTextInput(&appContext);
     initMap(appContext.map);
 
     if(clickListeners == NULL)
```

Comment:
This test should be right after the call to initClickListeners() (line 8) and include `fprintf(stderr, "malloc error");`